app/controllers/local_authorities_controller.rb

```diff
@@ -32,9 +32,7 @@ def filtered_links
 
     case params[:filter]
     when 'broken_links'
-      links.currently_broken
-    when 'good_links'
-      links.good_links
+      links.broken_or_missing
```

Comment:
Seems there's a repetition of code here, we could extract this into another helper class to receive the `params[:filter]` and return the "right thing"